src/Foundation/Commerce/code/Models/Mappings/UserProfile.cs

```diff
@@ -0,0 +1,68 @@
+﻿//    Copyright 2020 EPAM Systems, Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+namespace Wooli.Foundation.Commerce.Models.Mappings
+{
+    using System.Linq;
+
+    using AutoMapper;
+
+    using Entities.Users;
+
+    using Foundation.Account.Infrastructure.Pipelines.Login;
+    using Sitecore.Commerce.Entities.Customers;
+
+    using Constants = Utils.Constants;
+
+    public class UserProfile : Profile
+    {
+        public UserProfile()
+        {
+            this.CreateMap<User, LoginPipelineArgs>()
+                .ForMember(dest => dest.UserName, opt => opt.MapFrom(src => src.UserName))
+                .ForMember(dest => dest.ContactId, opt => opt.MapFrom(src => src.ContactId))
+                .ForMember(dest => dest.Email, opt => opt.MapFrom(src => src.Email))
+                .ForMember(dest => dest.CustomerId, opt => opt.MapFrom(src => src.CustomerId))
+                .ForMember(dest => dest.FirstName, opt => opt.MapFrom(src => src.FirstName))
+                .ForMember(dest => dest.LastName, opt => opt.MapFrom(src => src.LastName))
+                .ForAllOtherMembers(opt => opt.Ignore());
+
+            this.CreateMap<LoginPipelineArgs, User>()
```

Comment:
Is that a `.ReverseMap()` ?

Reply:
We need to ignore some fields in both directions, so `ReverseMap` not work correctly

Reply:
ok